Description: Exclude Xen dom0 from guest list
 With newer versions of libvirt Domain-0 is again visible in the list of
 running guests but it should not be considered as a guest for shutdown
 or suspend.
Forwarded: no
Author: Stefan Bader <stefan.bader@canonical.com>

Index: libvirt-1.3.0/tools/libvirt-guests.sh.in
===================================================================
--- libvirt-1.3.0.orig/tools/libvirt-guests.sh.in	2016-01-27 14:18:28.000000000 +0100
+++ libvirt-1.3.0/tools/libvirt-guests.sh.in	2016-01-27 14:18:59.215412907 +0100
@@ -133,7 +133,7 @@ list_guests() {
         return 1
     fi
 
-    echo $list
+    echo "$list" | grep -v 00000000-0000-0000-0000-000000000000
 }
 
 # guest_name URI UUID
@@ -551,7 +551,7 @@ gueststatus() {
     for uri in $URIS; do
         set +f
         echo "* $uri URI:"
-        retval run_virsh "$uri" list || echo
+        retval run_virsh "$uri" list | grep -v "Domain-0" || echo
     done
     set +f
 }
